server/chat-commands/info.ts

```diff
@@ -42,7 +42,9 @@ export const commands: ChatCommands = {
 		}
 		if (!targetUser.connected) buf += ` <em style="color:gray">(offline)</em>`;
 		let roomauth = '';
-		if (usedRoom.auth && targetUser.id in usedRoom.auth) roomauth = usedRoom.auth[targetUser.id];
+		if (usedRoom.auth.has(targetUser.id)) {
+			roomauth = usedRoom.auth.get(targetUser.id) as string;
```

Comment:
Why is this a cast?

Reply:
Oh, it returns `string | undefined`. It could just be `get(...)!`, but I would probably recommend making it default to returning `''`.

Reply:
Which would simplify this (and also line 253) to `const roomauth = userRoom.auth.get(targetUser.id)`

Reply:
Typescript also won't allow this. 
```ts
	get(id: ID) {
		if (!super.has(id)) return '';
		return super.get(id);
	}````


```Property 'get' in type 'Auth' is not assignable to the same property in base type 'Map<ID, GroupSymbol>'.
  Type '(id: ID) => "" | "~" | "&" | "#" | "★" | "*" | "@" | "%" | "☆" | "+" | " " | "‽" | "!" | undefined' is not assignable to type '(key: ID) => "~" | "&" | "#" | "★" | "*" | "@" | "%" | "☆" | "+" | " " | "‽" | "!" | undefined'.
    Type '"" | "~" | "&" | "#" | "★" | "*" | "@" | "%" | "☆" | "+" | " " | "‽" | "!" | undefined' is not assignable to type '"~" | "&" | "#" | "★" | "*" | "@" | "%" | "☆" | "+" | " " | "‽" | "!" | undefined'.
      Type '""' is not assignable to type '"~" | "&" | "#" | "★" | "*" | "@" | "%" | "☆" | "+" | " " | "‽" | "!" | undefined'.```

Reply:
I can change it to `get(...)!` though.